SEED-platform/seed
# 3392 filter error between inventory lists

#### Any background context you want to provide?
Filters of `is not None` break on the new inventory list.

#### What's this PR do?
Fixes filtering for `is not None` for default canonical columns

#### How should this be manually tested?
Go to inventory legacy and search filter a column that has empty values using `!=""` (double quotes, not single)
Navigate to inventory list and confirm there are no errors

#### What are the relevant tickets?
#3392


#### Screenshots (if appropriate)
![Screen Shot 2022-07-25 at 2 58 32 PM](https://user-images.githubusercontent.com/58446472/180872937-9a5186d5-03c8-429d-b01c-bda8c7c163e4.png)
